cognite/client/data_classes/transformations/common.py

```diff
@@ -290,6 +292,18 @@ def __init__(
         self.audience = audience
         self.cdf_project_name = cdf_project_name
 
+    def as_valid_credentials(self) -> OAuthClientCredentials:
```

Comment:
as_credential_provider maybe? The credentials are presumably already valid.

Reply:
That's the one! I swear, naming this method took me 95% of the time with this PR...